### PR TITLE
Page title and sub home

### PR DESCRIPTION
**Pull request by Ankita-Manmoandas**
added component, styling, and tests

### CODE REVIEW

**Comment by Ankita-Manmoandas** [comment]
component, sass, and tests complete

File: .eslintrc.json [hunk 1]
```diff
@@ -29,5 +29,6 @@
     "react": {
       "version": "detect"
     }
-  }
-}
+  }}
+  
+
```
Comment (DanForder) [inline .eslintrc.json:34]: undo this change

File: src/App.jsx [hunk 2]
```diff
@@ -6,12 +6,18 @@ import Schedule from "./pages/Schedule/Schedule";
 
 const App = () => {
   return (
+    <div>
```
Comment (DanForder) [inline src/App.jsx:9]: Remove this wrapping div, it's not being used

File: src/assets/sass/_typography.css [hunk 3]
```diff
@@ -0,0 +1 @@
+@import"https://fonts.googleapis.com/css2?family=PT+Sans&family=Poppins:wght@400;500;600;700&display=swap";@import"https://fonts.googleapis.com/css2?family=Lato&family=Playfair+Display:wght@400;500;600;700&display=swap";@import"https://fonts.googleapis.com/css2?family=Oswald:wght@300;400&display=swap"
```
Comment (DanForder) [inline src/assets/sass/_typography.css:1]: Remove this css file

File: src/assets/sass/_typography.scss [hunk 4]
```diff
@@ -1,8 +1,10 @@
 @import url("https://fonts.googleapis.com/css2?family=PT+Sans&family=Poppins:wght@400;500;600;700&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Lato&family=Playfair+Display:wght@400;500;600;700&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@300;400&display=swap');
 // Modern Fonts
 $modern-content: "PT Sans", sans-serif;
 $modern-header: "Poppins", sans-serif;
+$modern-header-sub:'Oswald', sans-serif;
```
Comment (DanForder) [inline src/assets/sass/_typography.scss:7]: this font is the same irrespective of modern/ classic theme, so can just be something like $sub-heading-font

**Review by DanForder — CHANGES_REQUESTED** [review]

**Comment by DanForder** [comment]
Please run prettier with NPM run prettier

**Comment by DanForder** [comment]
Remove CSS files, we're using SCSS in the project

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/EventHeaderTitle/EventHeaderTitle.jsx [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
-  return <div className="header-home">
-    <h1 className="header-home__title">{title}</h1>
-    <h1 className="header-home__subtitle">{subtitle}</h1>
-  </div>
-}
+const EventHeaderTitle = (props) => {
+  const { title, subtitle } = props;
+  return (
+    <div className="header-home">
+      <span className="header-home__title">{title}</span>
```
Comment (DanForder) [inline src/components/EventHeaderTitle/EventHeaderTitle.jsx:7]: This one should still be an H1, just the subtitle needs to be span. We just want to make sure there's always exactly one H1 on each page

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/ViewScheduleButton/ViewScheduleButton.css [hunk 6]
```diff
@@ -0,0 +1 @@
+@import"https://fonts.googleapis.com/css2?family=PT+Sans&family=Poppins:wght@400;500;600;700&display=swap";@import"https://fonts.googleapis.com/css2?family=Lato&family=Playfair+Display:wght@400;500;600;700&display=swap";.view-schedule-button{height:66px;width:335px;text-transform:uppercase;background-color:#3c3c3c;border-radius:5px;color:#fff;font-weight:700;font-family:"Poppins",sans-serif;font-size:1rem;display:flex;flex-direction:column;justify-content:center;align-items:center;text-decoration:none}
```
Comment (DanForder) [inline src/components/ViewScheduleButton/ViewScheduleButton.css:1]: remove css file

**Review by DanForder — CHANGES_REQUESTED** [review]

**Pull request merged**